canonical-websites/jaas.ai
# Add basic search

## Done
- added search box to header
- added search page including recommended results

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- do a search from the header
- you should see recommended results
- clicking the name of a charm/bundle should take you to the charm/bundle

## Details
- Fixes: #53.
- Fixes: #54.
